src/MysqlClient.php

```diff
@@ -361,6 +361,11 @@ function (\Exception $e) {
      * $mysql->quit();
      * ```
      *
+     * This method will gracefully close the connection to the MySQL database
+     * server once all outstanding commands are completed. See also
+     * [`close()`)(#close) if you want to force-close the connection without
```

Comment:
Nit: I think you used a wrong closing bracket here

Reply:
Thanks for spotting, updated as suggested :shipit: